.eslintignore

```diff
@@ -1,8 +1,8 @@
 build/
 dist/
 node_modules/
+docs/
```

Comment:
In docs folder was several eslint error, so in another pull request I should integrate it with shared eslint config